lib/spack/spack/build_systems/autotools.py

```diff
@@ -184,12 +189,6 @@ def patch(self):
             ``config.guess``
         """
```

Comment:
I did a test removing the patch() function and it appears to work OK to me, but my tests were by no means exhaustive. Should I leave the empty patch(), put a "pass" in there, or remove the patch() function?